### DISCUSSION TITLE
Save custom views or easy filter by category

### DISCUSSION THREAD

**Discussion by KGAIBERIA** [snipe/snipe-it#14978] [Ideas]
I have different assets categories: laptops, smartphones, SIM cards, etc... 

Each category can have a different fieldset assigned with different custom fields. For example, smartphones has IMEIs field, and SIM cards have PIN, PUK or Telephone Number.

Would be great to have the possibility to save custom views. For example, when I am checking only category smartphones I need to see fields that I do not need when I am checking laptops. Custom view should save at least columns order and would be fantastic if also save applied filters (for example category = smartphone). When save a custom view you can rename it for easy selection on a view dropdown menu.

An easy access for filtering categories would be also an easier posibility.

I tried to search a similar idea but I did not found any. If exist, please let me know to vote it.

If you have a solution to my idea please, let me know how you work with Snipe-IT. **I want to hide unwanted columms because if all colums cannot be displayed on the screen, move to left and right to view latest columns is very dificult on snipe-it and the view in full screen mode with lot of columms is also impossible.**

Congratulations for your fantastic project

**Comment by Santrexxx** [comment]
Really, we need to make it possible to create "Custom" views for each Category or Model.
Because what's the point of Custom Fields if it's completely impossible to use them.
Moreover, we need "Template" options that the Admin can prepare in advance. Afterwards, the Administrator can assign them to Users as "Default" (so that they don't have to dig through these endless columns).

**Comment by Santrexxx** [comment]
I also need Custom views with Filters for Assets that I could save and call by buttons. Since selecting from 1000+ objects is complex, and starting a complex search every time is a headache.
